Tests/TestingTests/TestSupport/TestingAdditions.swift

```diff
@@ -290,6 +290,10 @@ extension Test.ID.Selection {
   }
 }
 
+func makeTestFilter(matching selection: Test.ID.Selection) -> Configuration.TestFilter {
```

Comment:
DocC!

Reply:
This file doesn't need docc stuff, it's just test fixtures!